Example/Glia Sample Integration/Presenters/ContentViewPresenter.swift

```diff
@@ -0,0 +1,117 @@
+import SwiftUI
+import SalemoveSDK
+import GliaWidgets
+
+class ContentViewPresenter: ObservableObject {
+
+    private let config: GliaConfiguration
```

Comment:
This file has a lot of configurations between GliaConfiguration, Configuration, and Salemove.Configuration, so this variable would benefit from having a descriptive name, like `gliaConfiguration`, even if it's longer.

Reply:
Fixed [af4a31c](https://github.com/salemove/ios-sdk-widgets/commit/af4a31cbdfdff9ae7acfa5eb81c2eaee841033d5)